wit-manifest.json

```diff
@@ -248,5 +248,10 @@
         "commit": "478f67af71b126e47cfb0839c6238c07f7e2bb2b",
         "name": "example-buserror",
         "source": "git@github.com:sifive/example-buserror.git"
+    },
+    {
+        "commit": "9a6c54b7bd620e61e88cc79a157057eb9b625b29",
+        "name": "example-hwperfmon",
+        "source": "git@github.com:sifive/example-hwperfmon.git"
```

Comment:
Some comments for the example.

1. Best to have the repo name as "example-hardware-performance-monitor"
2. Should drop the the timestamp print, to ensure it fit many design.
3. In addition/instead of printing out message, the example should return non-zero code for failure
4. The example should also be cable of running in RTL environment, so it need to exit.

Reply:
Fixed.